presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -94,6 +95,9 @@ public String explain(CreateTable statement, List<Expression> parameters)
                 if ((type == null) || type.equals(UNKNOWN)) {
                     throw new SemanticException(TYPE_MISMATCH, column, "Unknown type for column '%s' ", column.getName());
                 }
+                if (columns.stream().anyMatch(c -> c.getName().equals(column.getName()))) {
```

Comment:
I'm pretty sure this is not a big deal, but such O(n2) doesn't look good to me. What about turning columns into `LinkedHashMap<String /*name*/, ColumnMetadata>` ?